src/Mutation/PostObjectCreate.php

```diff
@@ -211,8 +212,16 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
             /**
              * If the post being created is being assigned to another user that's not the current user, make sure
              * the current user has permission to edit others posts for this post_type
-             */
-            if ( ! empty( $input['authorId'] ) && get_current_user_id() !== $input['authorId'] && ! current_user_can( $post_type_object->cap->edit_others_posts ) ) {
+             *
+	     *
+	     * Bug Fix: ID failed if it was user id in string format i.e. '120029. Need to account for Hash/Gloabl ID's as well as numeric.
+	     */
+	    $authorID = absint($input['authorId'])
```

Comment:
looks like there's a syntax error in here causing all tests to fail. 

I think the missing `;` on line 219 is an issue for sure

Then spacing on line 220 also needs adjusted

Reply:
@joseffb-mla (mentioning you to make sure you get notified)